Cargo.toml

```diff
@@ -21,24 +21,21 @@ is-it-maintained-open-issues = { repository = "mozilla/mtu", branch = "main" }
 maintenance = { status = "actively-developed", branch = "main" }
 
 [dependencies]
-# Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
-libc = { version = "0.2", default-features = false }
-static_assertions = { version = "1.1", default-features = false }
+libc = { version = "0", default-features = false }
+static_assertions = { version = "1", default-features = false }
```

Comment:
I assume that we don't make use of any `static_assertions` features that are only available since `1.1`, i.e. not available in `1.0`? If so, the above is correct.